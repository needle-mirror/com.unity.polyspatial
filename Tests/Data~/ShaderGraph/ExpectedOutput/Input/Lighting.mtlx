<?xml version ="1.0"?>
<materialx version = "1.38" colorspace = "lin_rec709">
    <dot name = "_1" type = "vector4">
        <input name = "in" type = "vector4" nodename = "unity_AmbientSky" />
    </dot>
    <constant name = "unity_AmbientSky" type = "vector4">
        <input name = "value" type = "vector4" value = "0,0,0,0" />
    </constant>
    <dot name = "_2" type = "vector4">
        <input name = "in" type = "vector4" nodename = "unity_AmbientEquator" />
    </dot>
    <constant name = "unity_AmbientEquator" type = "vector4">
        <input name = "value" type = "vector4" value = "0,0,0,0" />
    </constant>
    <dot name = "_3" type = "vector4">
        <input name = "in" type = "vector4" nodename = "unity_AmbientGround" />
    </dot>
    <constant name = "unity_AmbientGround" type = "vector4">
        <input name = "value" type = "vector4" value = "0,0,0,0" />
    </constant>
    <dot name = "_4" type = "vector3">
        <input name = "in" type = "vector3" nodename = "_5" />
    </dot>
    <ifequal name = "_5" type = "vector3">
        <input name = "value1" type = "float" nodename = "_6" />
        <input name = "value2" type = "float" value = "0" />
        <input name = "in1" type = "vector3" nodename = "_7" />
        <input name = "in2" type = "vector3" nodename = "_37" />
    </ifequal>
    <convert name = "_6" type = "float">
        <input name = "in" type = "boolean" nodename = "LIGHTMAP_ON" />
    </convert>
    <constant name = "LIGHTMAP_ON" type = "boolean">
        <input name = "value" type = "boolean" value = "false" />
    </constant>
    <max name = "_7" type = "vector3">
        <input name = "in1" type = "vector3" value = "0,0,0" />
        <input name = "in2" type = "vector3" nodename = "_8" />
    </max>
    <add name = "_8" type = "vector3">
        <input name = "in1" type = "vector3" nodename = "_9" />
        <input name = "in2" type = "vector3" nodename = "_28" />
    </add>
    <add name = "_9" type = "vector3">
        <input name = "in1" type = "vector3" nodename = "_10" />
        <input name = "in2" type = "vector3" nodename = "_21" />
    </add>
    <combine3 name = "_10" type = "vector3">
        <input name = "in1" type = "float" nodename = "_11" />
        <input name = "in2" type = "float" nodename = "_19" />
        <input name = "in3" type = "float" nodename = "_20" />
    </combine3>
    <dotproduct name = "_11" type = "float">
        <input name = "in1" type = "vector4" nodename = "polySpatial_SHAr" />
        <input name = "in2" type = "vector4" nodename = "_12" />
    </dotproduct>
    <constant name = "polySpatial_SHAr" type = "vector4">
        <input name = "value" type = "vector4" value = "0,0,0,0" />
    </constant>
    <combine4 name = "_12" type = "vector4">
        <input name = "in1" type = "float" nodename = "_13" />
        <input name = "in2" type = "float" nodename = "_17" />
        <input name = "in3" type = "float" nodename = "_18" />
        <input name = "in4" type = "float" value = "1" />
    </combine4>
    <swizzle name = "_13" type = "float">
        <input name = "in" type = "vector3" nodename = "_14" />
        <input name = "channels" type = "string" value = "x" />
    </swizzle>
    <dot name = "_14" type = "vector3">
        <input name = "in" type = "vector3" nodename = "_16" />
    </dot>
    <normal name = "_15" type = "vector3">
        <input name = "space" type = "string" value = "world" />
    </normal>
    <multiply name = "_16" type = "vector3">
        <input name = "in1" type = "vector3" nodename = "_15" />
        <input name = "in2" type = "vector3" value = "1,1,-1" />
    </multiply>
    <swizzle name = "_17" type = "float">
        <input name = "in" type = "vector3" nodename = "_14" />
        <input name = "channels" type = "string" value = "y" />
    </swizzle>
    <swizzle name = "_18" type = "float">
        <input name = "in" type = "vector3" nodename = "_14" />
        <input name = "channels" type = "string" value = "z" />
    </swizzle>
    <dotproduct name = "_19" type = "float">
        <input name = "in1" type = "vector4" nodename = "polySpatial_SHAg" />
        <input name = "in2" type = "vector4" nodename = "_12" />
    </dotproduct>
    <constant name = "polySpatial_SHAg" type = "vector4">
        <input name = "value" type = "vector4" value = "0,0,0,0" />
    </constant>
    <dotproduct name = "_20" type = "float">
        <input name = "in1" type = "vector4" nodename = "polySpatial_SHAb" />
        <input name = "in2" type = "vector4" nodename = "_12" />
    </dotproduct>
    <constant name = "polySpatial_SHAb" type = "vector4">
        <input name = "value" type = "vector4" value = "0,0,0,0" />
    </constant>
    <combine3 name = "_21" type = "vector3">
        <input name = "in1" type = "float" nodename = "_22" />
        <input name = "in2" type = "float" nodename = "_26" />
        <input name = "in3" type = "float" nodename = "_27" />
    </combine3>
    <dotproduct name = "_22" type = "float">
        <input name = "in1" type = "vector4" nodename = "polySpatial_SHBr" />
        <input name = "in2" type = "vector4" nodename = "_23" />
    </dotproduct>
    <constant name = "polySpatial_SHBr" type = "vector4">
        <input name = "value" type = "vector4" value = "0,0,0,0" />
    </constant>
    <multiply name = "_23" type = "vector4">
        <input name = "in1" type = "vector4" nodename = "_24" />
        <input name = "in2" type = "vector4" nodename = "_25" />
    </multiply>
    <swizzle name = "_24" type = "vector4">
        <input name = "in" type = "vector3" nodename = "_14" />
        <input name = "channels" type = "string" value = "xyzz" />
    </swizzle>
    <swizzle name = "_25" type = "vector4">
        <input name = "in" type = "vector3" nodename = "_14" />
        <input name = "channels" type = "string" value = "yzzx" />
    </swizzle>
    <dotproduct name = "_26" type = "float">
        <input name = "in1" type = "vector4" nodename = "polySpatial_SHBg" />
        <input name = "in2" type = "vector4" nodename = "_23" />
    </dotproduct>
    <constant name = "polySpatial_SHBg" type = "vector4">
        <input name = "value" type = "vector4" value = "0,0,0,0" />
    </constant>
    <dotproduct name = "_27" type = "float">
        <input name = "in1" type = "vector4" nodename = "polySpatial_SHBb" />
        <input name = "in2" type = "vector4" nodename = "_23" />
    </dotproduct>
    <constant name = "polySpatial_SHBb" type = "vector4">
        <input name = "value" type = "vector4" value = "0,0,0,0" />
    </constant>
    <multiply name = "_28" type = "vector3">
        <input name = "in1" type = "vector3" nodename = "_29" />
        <input name = "in2" type = "float" nodename = "_30" />
    </multiply>
    <swizzle name = "_29" type = "vector3">
        <input name = "in" type = "vector4" nodename = "polySpatial_SHC" />
        <input name = "channels" type = "string" value = "xyz" />
    </swizzle>
    <constant name = "polySpatial_SHC" type = "vector4">
        <input name = "value" type = "vector4" value = "0,0,0,0" />
    </constant>
    <subtract name = "_30" type = "float">
        <input name = "in1" type = "float" nodename = "_31" />
        <input name = "in2" type = "float" nodename = "_34" />
    </subtract>
    <multiply name = "_31" type = "float">
        <input name = "in1" type = "float" nodename = "_32" />
        <input name = "in2" type = "float" nodename = "_33" />
    </multiply>
    <swizzle name = "_32" type = "float">
        <input name = "in" type = "vector3" nodename = "_14" />
        <input name = "channels" type = "string" value = "x" />
    </swizzle>
    <swizzle name = "_33" type = "float">
        <input name = "in" type = "vector3" nodename = "_14" />
        <input name = "channels" type = "string" value = "x" />
    </swizzle>
    <multiply name = "_34" type = "float">
        <input name = "in1" type = "float" nodename = "_35" />
        <input name = "in2" type = "float" nodename = "_36" />
    </multiply>
    <swizzle name = "_35" type = "float">
        <input name = "in" type = "vector3" nodename = "_14" />
        <input name = "channels" type = "string" value = "y" />
    </swizzle>
    <swizzle name = "_36" type = "float">
        <input name = "in" type = "vector3" nodename = "_14" />
        <input name = "channels" type = "string" value = "y" />
    </swizzle>
    <divide name = "_37" type = "vector3">
        <input name = "in1" type = "vector3" nodename = "_38" />
        <input name = "in2" type = "float" nodename = "_55" />
    </divide>
    <multiply name = "_38" type = "vector3">
        <input name = "in1" type = "vector3" nodename = "_39" />
        <input name = "in2" type = "float" nodename = "_50" />
    </multiply>
    <multiply name = "_39" type = "vector3">
        <input name = "in1" type = "vector3" nodename = "_40" />
        <input name = "in2" type = "float" value = "4.59" />
    </multiply>
    <swizzle name = "_40" type = "vector3">
        <input name = "in" type = "vector4" nodename = "_41" />
        <input name = "channels" type = "string" value = "xyz" />
    </swizzle>
    <RealityKitTexture2D name = "_41" type = "vector4">
        <input name = "file" type = "filename" nodename = "polySpatial_Lightmap" />
        <input name = "texcoord" type = "vector2" nodename = "_42" />
        <input name = "min_filter" type = "string" value = "linear" />
        <input name = "mag_filter" type = "string" value = "linear" />
        <input name = "mip_filter" type = "string" value = "nearest" />
        <input name = "u_wrap_mode" type = "string" value = "clamp_to_edge" />
        <input name = "v_wrap_mode" type = "string" value = "clamp_to_edge" />
        <input name = "max_anisotropy" type = "integer" value = "1" />
    </RealityKitTexture2D>
    <constant name = "polySpatial_Lightmap" type = "filename">
        <input name = "value" type = "filename" value = "ERR" />
    </constant>
    <add name = "_42" type = "vector2">
        <input name = "in1" type = "vector2" nodename = "_43" />
        <input name = "in2" type = "vector2" nodename = "_49" />
    </add>
    <multiply name = "_43" type = "vector2">
        <input name = "in1" type = "vector2" nodename = "_44" />
        <input name = "in2" type = "vector2" nodename = "_48" />
    </multiply>
    <dot name = "_44" type = "vector2">
        <input name = "in" type = "vector2" nodename = "_47" />
    </dot>
    <UsdPrimvarReader name = "_45" type = "vector2">
        <input name = "varname" type = "string" value = "vertexUV1" />
    </UsdPrimvarReader>
    <multiply name = "_46" type = "vector2">
        <input name = "in1" type = "vector2" nodename = "_45" />
        <input name = "in2" type = "vector2" value = "1,-1" />
    </multiply>
    <add name = "_47" type = "vector2">
        <input name = "in1" type = "vector2" nodename = "_46" />
        <input name = "in2" type = "vector2" value = "0,1" />
    </add>
    <swizzle name = "_48" type = "vector2">
        <input name = "in" type = "vector4" nodename = "polySpatial_LightmapST" />
        <input name = "channels" type = "string" value = "xy" />
    </swizzle>
    <constant name = "polySpatial_LightmapST" type = "vector4">
        <input name = "value" type = "vector4" value = "0,0,0,0" />
    </constant>
    <swizzle name = "_49" type = "vector2">
        <input name = "in" type = "vector4" nodename = "polySpatial_LightmapST" />
        <input name = "channels" type = "string" value = "zw" />
    </swizzle>
    <add name = "_50" type = "float">
        <input name = "in1" type = "float" nodename = "_51" />
        <input name = "in2" type = "float" value = "0.5" />
    </add>
    <dotproduct name = "_51" type = "float">
        <input name = "in1" type = "vector3" nodename = "_14" />
        <input name = "in2" type = "vector3" nodename = "_52" />
    </dotproduct>
    <subtract name = "_52" type = "vector3">
        <input name = "in1" type = "vector3" nodename = "_53" />
        <input name = "in2" type = "float" value = "0.5" />
    </subtract>
    <swizzle name = "_53" type = "vector3">
        <input name = "in" type = "vector4" nodename = "_54" />
        <input name = "channels" type = "string" value = "xyz" />
    </swizzle>
    <RealityKitTexture2D name = "_54" type = "vector4">
        <input name = "file" type = "filename" nodename = "polySpatial_LightmapInd" />
        <input name = "texcoord" type = "vector2" nodename = "_42" />
        <input name = "min_filter" type = "string" value = "linear" />
        <input name = "mag_filter" type = "string" value = "linear" />
        <input name = "mip_filter" type = "string" value = "nearest" />
        <input name = "u_wrap_mode" type = "string" value = "clamp_to_edge" />
        <input name = "v_wrap_mode" type = "string" value = "clamp_to_edge" />
        <input name = "max_anisotropy" type = "integer" value = "1" />
    </RealityKitTexture2D>
    <constant name = "polySpatial_LightmapInd" type = "filename">
        <input name = "value" type = "filename" value = "ERR" />
    </constant>
    <max name = "_55" type = "float">
        <input name = "in1" type = "float" value = "0.0001" />
        <input name = "in2" type = "float" nodename = "_56" />
    </max>
    <swizzle name = "_56" type = "float">
        <input name = "in" type = "vector4" nodename = "_54" />
        <input name = "channels" type = "string" value = "w" />
    </swizzle>
    <add name = "_57" type = "vector3">
        <input name = "in1" type = "vector3" nodename = "_105" />
        <input name = "in2" type = "vector3" nodename = "_102" />
    </add>
    <dot name = "_58" type = "vector3">
        <input name = "in" type = "vector3" nodename = "_59" />
    </dot>
    <multiply name = "_59" type = "vector3">
        <input name = "in1" type = "vector3" nodename = "_60" />
        <input name = "in2" type = "float" value = "4.59" />
    </multiply>
    <swizzle name = "_60" type = "vector3">
        <input name = "in" type = "vector4" nodename = "_61" />
        <input name = "channels" type = "string" value = "xyz" />
    </swizzle>
    <RealityKitTexture2DLOD name = "_61" type = "vector4">
        <input name = "file" type = "filename" nodename = "polySpatial_SpecCube0" />
        <input name = "texcoord" type = "vector2" nodename = "_62" />
        <input name = "lod" type = "float" nodename = "_101" />
        <input name = "u_wrap_mode" type = "string" value = "clamp_to_edge" />
        <input name = "v_wrap_mode" type = "string" value = "clamp_to_edge" />
        <input name = "min_filter" type = "string" value = "linear" />
        <input name = "mag_filter" type = "string" value = "linear" />
        <input name = "mip_filter" type = "string" value = "nearest" />
        <input name = "max_anisotropy" type = "integer" value = "1" />
    </RealityKitTexture2DLOD>
    <constant name = "polySpatial_SpecCube0" type = "filename">
        <input name = "value" type = "filename" value = "ERR" />
    </constant>
    <ifgreater name = "_62" type = "vector2">
        <input name = "value1" type = "float" nodename = "_63" />
        <input name = "value2" type = "float" nodename = "_73" />
        <input name = "in1" type = "vector2" nodename = "_74" />
        <input name = "in2" type = "vector2" nodename = "_92" />
    </ifgreater>
    <swizzle name = "_63" type = "float">
        <input name = "in" type = "vector3" nodename = "_64" />
        <input name = "channels" type = "string" value = "x" />
    </swizzle>
    <absval name = "_64" type = "vector3">
        <input name = "in" type = "vector3" nodename = "_65" />
    </absval>
    <realitykit_reflect name = "_65" type = "vector3">
        <input name = "in" type = "vector3" nodename = "_66" />
        <input name = "normal" type = "vector3" nodename = "_70" />
    </realitykit_reflect>
    <subtract name = "_66" type = "vector3">
        <input name = "in2" type = "vector3" nodename = "_67" />
    </subtract>
    <dot name = "_67" type = "vector3">
        <input name = "in" type = "vector3" nodename = "_69" />
    </dot>
    <realitykit_viewdirection name = "_68" type = "vector3">
    </realitykit_viewdirection>
    <multiply name = "_69" type = "vector3">
        <input name = "in1" type = "vector3" nodename = "_68" />
        <input name = "in2" type = "vector3" value = "1,1,-1" />
    </multiply>
    <dot name = "_70" type = "vector3">
        <input name = "in" type = "vector3" nodename = "_72" />
    </dot>
    <normal name = "_71" type = "vector3">
        <input name = "space" type = "string" value = "object" />
    </normal>
    <multiply name = "_72" type = "vector3">
        <input name = "in1" type = "vector3" nodename = "_71" />
        <input name = "in2" type = "vector3" value = "1,1,-1" />
    </multiply>
    <swizzle name = "_73" type = "float">
        <input name = "in" type = "vector3" nodename = "_64" />
        <input name = "channels" type = "string" value = "y" />
    </swizzle>
    <ifgreater name = "_74" type = "vector2">
        <input name = "value1" type = "float" nodename = "_63" />
        <input name = "value2" type = "float" nodename = "_75" />
        <input name = "in1" type = "vector2" nodename = "_76" />
        <input name = "in2" type = "vector2" nodename = "_84" />
    </ifgreater>
    <swizzle name = "_75" type = "float">
        <input name = "in" type = "vector3" nodename = "_64" />
        <input name = "channels" type = "string" value = "z" />
    </swizzle>
    <ifgreater name = "_76" type = "vector2">
        <input name = "value1" type = "float" nodename = "_77" />
        <input name = "value2" type = "float" value = "0" />
        <input name = "in1" type = "vector2" nodename = "_78" />
        <input name = "in2" type = "vector2" nodename = "_82" />
    </ifgreater>
    <swizzle name = "_77" type = "float">
        <input name = "in" type = "vector3" nodename = "_65" />
        <input name = "channels" type = "string" value = "x" />
    </swizzle>
    <add name = "_78" type = "vector2">
        <input name = "in1" type = "vector2" nodename = "_79" />
        <input name = "in2" type = "vector2" value = "0.5,0.08333334" />
    </add>
    <multiply name = "_79" type = "vector2">
        <input name = "in1" type = "vector2" nodename = "_80" />
        <input name = "in2" type = "vector2" value = "-0.5,-0.08333334" />
    </multiply>
    <divide name = "_80" type = "vector2">
        <input name = "in1" type = "vector2" nodename = "_81" />
        <input name = "in2" type = "float" nodename = "_63" />
    </divide>
    <swizzle name = "_81" type = "vector2">
        <input name = "in" type = "vector3" nodename = "_65" />
        <input name = "channels" type = "string" value = "zy" />
    </swizzle>
    <add name = "_82" type = "vector2">
        <input name = "in1" type = "vector2" nodename = "_83" />
        <input name = "in2" type = "vector2" value = "0.5,0.25" />
    </add>
    <multiply name = "_83" type = "vector2">
        <input name = "in1" type = "vector2" nodename = "_80" />
        <input name = "in2" type = "vector2" value = "0.5,-0.08333334" />
    </multiply>
    <ifgreater name = "_84" type = "vector2">
        <input name = "value1" type = "float" nodename = "_85" />
        <input name = "value2" type = "float" value = "0" />
        <input name = "in1" type = "vector2" nodename = "_86" />
        <input name = "in2" type = "vector2" nodename = "_90" />
    </ifgreater>
    <swizzle name = "_85" type = "float">
        <input name = "in" type = "vector3" nodename = "_65" />
        <input name = "channels" type = "string" value = "z" />
    </swizzle>
    <add name = "_86" type = "vector2">
        <input name = "in1" type = "vector2" nodename = "_87" />
        <input name = "in2" type = "vector2" value = "0.5,0.75" />
    </add>
    <multiply name = "_87" type = "vector2">
        <input name = "in1" type = "vector2" nodename = "_88" />
        <input name = "in2" type = "vector2" value = "0.5,-0.08333334" />
    </multiply>
    <divide name = "_88" type = "vector2">
        <input name = "in1" type = "vector2" nodename = "_89" />
        <input name = "in2" type = "float" nodename = "_75" />
    </divide>
    <swizzle name = "_89" type = "vector2">
        <input name = "in" type = "vector3" nodename = "_65" />
        <input name = "channels" type = "string" value = "xy" />
    </swizzle>
    <add name = "_90" type = "vector2">
        <input name = "in1" type = "vector2" nodename = "_91" />
        <input name = "in2" type = "vector2" value = "0.5,0.9166667" />
    </add>
    <multiply name = "_91" type = "vector2">
        <input name = "in1" type = "vector2" nodename = "_88" />
        <input name = "in2" type = "vector2" value = "-0.5,-0.08333334" />
    </multiply>
    <ifgreater name = "_92" type = "vector2">
        <input name = "value1" type = "float" nodename = "_73" />
        <input name = "value2" type = "float" nodename = "_75" />
        <input name = "in1" type = "vector2" nodename = "_93" />
        <input name = "in2" type = "vector2" nodename = "_84" />
    </ifgreater>
    <ifgreater name = "_93" type = "vector2">
        <input name = "value1" type = "float" nodename = "_94" />
        <input name = "value2" type = "float" value = "0" />
        <input name = "in1" type = "vector2" nodename = "_95" />
        <input name = "in2" type = "vector2" nodename = "_99" />
    </ifgreater>
    <swizzle name = "_94" type = "float">
        <input name = "in" type = "vector3" nodename = "_65" />
        <input name = "channels" type = "string" value = "y" />
    </swizzle>
    <add name = "_95" type = "vector2">
        <input name = "in1" type = "vector2" nodename = "_96" />
        <input name = "in2" type = "vector2" value = "0.5,0.4166667" />
    </add>
    <multiply name = "_96" type = "vector2">
        <input name = "in1" type = "vector2" nodename = "_97" />
        <input name = "in2" type = "vector2" value = "0.5,0.08333334" />
    </multiply>
    <divide name = "_97" type = "vector2">
        <input name = "in1" type = "vector2" nodename = "_98" />
        <input name = "in2" type = "float" nodename = "_73" />
    </divide>
    <swizzle name = "_98" type = "vector2">
        <input name = "in" type = "vector3" nodename = "_65" />
        <input name = "channels" type = "string" value = "xz" />
    </swizzle>
    <add name = "_99" type = "vector2">
        <input name = "in1" type = "vector2" nodename = "_100" />
        <input name = "in2" type = "vector2" value = "0.5,0.5833333" />
    </add>
    <multiply name = "_100" type = "vector2">
        <input name = "in1" type = "vector2" nodename = "_97" />
        <input name = "in2" type = "vector2" value = "0.5,-0.08333334" />
    </multiply>
    <dot name = "_101" type = "float">
        <input name = "in" type = "float" value = "0" />
    </dot>
    <add name = "_102" type = "vector3">
        <input name = "in1" type = "vector3" nodename = "_4" />
        <input name = "in2" type = "vector3" nodename = "_58" />
    </add>
    <GeometryModification name = "_103" type = "vertexshader">
    </GeometryModification>
    <realitykit_unlit name = "_104" type = "surfaceshader">
        <input name = "applyPostProcessToneMap" type = "boolean" value = "false" />
        <input name = "color" type = "color3" nodename = "_106" />
    </realitykit_unlit>
    <surfacematerial name = "USD_Lighting" type = "material">
        <input name = "vertexshader" type = "vertexshader" nodename = "_103" />
        <input name = "surfaceshader" type = "surfaceshader" nodename = "_104" />
    </surfacematerial>
    <swizzle name = "_105" type = "vector3">
        <input name = "in" type = "vector4" nodename = "_1" />
        <input name = "channels" type = "string" value = "xyz" />
    </swizzle>
    <swizzle name = "_106" type = "color3">
        <input name = "in" type = "vector3" nodename = "_57" />
        <input name = "channels" type = "string" value = "xyz" />
    </swizzle>
</materialx>
