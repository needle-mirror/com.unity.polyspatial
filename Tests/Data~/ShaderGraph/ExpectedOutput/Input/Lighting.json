{"properties":[{"name":"unity_AmbientSky","type":3},{"name":"unity_AmbientEquator","type":3},{"name":"unity_AmbientGround","type":3},{"name":"LIGHTMAP_ON","type":0},{"name":"polySpatial_SHAr","type":3},{"name":"polySpatial_SHAg","type":3},{"name":"polySpatial_SHAb","type":3},{"name":"polySpatial_SHBr","type":3},{"name":"polySpatial_SHBg","type":3},{"name":"polySpatial_SHBb","type":3},{"name":"polySpatial_SHC","type":3},{"name":"polySpatial_Lightmap","type":6},{"name":"polySpatial_LightmapST","type":3},{"name":"polySpatial_LightmapInd","type":6},{"name":"polySpatial_SpecCube0","type":6}]}